pkg/parca/parca_test.go

```diff
@@ -295,46 +295,43 @@ func TestConsistency(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, f.Close())
 
-	for _, s := range p1.Sample {
-		s.Label = nil
-		s.NumLabel = nil
-		s.NumUnit = nil
-	}
-
 	p1 = p1.Compact()
 
 	p, err := parcaprofile.FromPprof(ctx, logger, m, p1, 0, false)
 	require.NoError(t, err)
 
-	_, err = parcacol.InsertProfileIntoTable(ctx, logger, table, labels.Labels{{}}, p)
+	_, err = parcacol.InsertProfileIntoTable(ctx, logger, table, labels.Labels{}, p)
 	require.NoError(t, err)
 
 	table.Sync()
 
-	api := queryservice.NewColumnQueryAPI(
-		logger,
-		tracer,
-		m,
-		query.NewEngine(
-			memory.DefaultAllocator,
-			colDB.TableProvider(),
-		),
-		"stacktraces",
+	q := query.NewEngine(
```

Comment:
Why replace the call that we expect a user to make with this lower-level call?

Reply:
Good question. This might have been a rebase mistake!